linux/library/src/internal/text_input_plugin.h

```diff
@@ -46,6 +46,9 @@ class TextInputPlugin : public KeyboardHookHandler, public JsonPlugin {
   // Sends the current state of the given model to the Flutter engine.
   void SendStateUpdate(const TextInputModel &model);
 
+  // Sends a "newline" action to the Flutter engine.
```

Comment:
Please document this as handling the Enter key rather than by the message it sends, since the latter could (and in this case, is very likely to) change.